CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Versions
 
 ## x.x.x
+* Fixed for Inter, Rewarded and AppOpen to load ads before the AppLovinMAX initialization.
```

Comment:
I'd keep consistency with how we reference these ad types in our public facing docs if we want to reference them. However, I think this is more descriptive of the issue:

```suggestion
* Allow API calls even before plugin had finished initialization. Previously, this would result in an error thrown or in older plugins, a no-op.
```